docs/gallery.rst

```diff
@@ -1,10 +1,7 @@
 .. -*- mode: rst -*-
 
-====================
-Example Gallery
-=======
 Gallery
-====================
+=======
```

Comment:
Not sure why this header was like it, but Sphinx was complaining so I updated it.